test/unit/command/test_status.py

```diff
@@ -24,12 +24,10 @@
 
 
 def test_execute(capsys, patched_main, molecule_instance):
-    molecule_instance.args = {'porcelain': False}
-
     s = status.Status({}, {}, molecule_instance)
     result = s.execute()
 
-    out, err = capsys.readouterr()
+    out, _ = capsys.readouterr()
```

Comment:
interesting, so, you're saying `capsys.readouterr()` is going to return two things, but I only care about the first one, right?